src/Core/BackendAggregator.vala

```diff
@@ -37,13 +39,25 @@ public class AppCenterCore.BackendAggregator : Backend, Object {
                         remove_inhibit_timeout = 0;
                     }
 
-                    SuspendControl.get_default ().inhibit ();
+                    if (inhibit_token != 0) {
+                        app.uninhibit (inhibit_token);
+                    }
```

Comment:
This seems to defeat the object of the uninhibit timeout.  By inserting logging messages and running appcenter in a terminal you can see that about a dozen inhibit/uninhibit pairs are produced in a short time while starting up. Why not just reset the timeout and keep the token without uninhibiting?

Reply:
Yeah I think you're right. Changed it to only add an inhibit if one does not already exist